international-journal-of-geriatric-psychiatry.csl

```diff
@@ -146,7 +145,7 @@
       </if>
     </choose>
   </macro>
-  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">
+  <citation delimiter-precedes-et-al="never" et-al-min="4" et-al-use-first="1" disambiguate-add-names="true" disambiguate-add-givenname="true" disambiguate-add-year-suffix="true" collapse="year">
```

Comment:
Looking at the Crellin et al. reference in http://onlinelibrary.wiley.com/doi/10.1002/gps.4447/epdf I think this should be 3/1